operator/bundle_helpers/requirements-build.txt

```diff
@@ -0,0 +1,19 @@
+attrs==24.2.0
```

Comment:
I did not know about `requirements-build.txt` before and it seems to be Cachi2 thing.

What was the reason you decided to build from sdists instead of using wheels (as described [here](https://github.com/containerbuildsystem/cachi2/blob/main/docs/pip.md#distribution-formats))?

If we stay with `requirements-build.txt`, we need instructions how to update it when we change versions in `requirements.txt` or add new dependencies there.

Reply:
I saw wheels support was just announced https://groups.google.com/a/redhat.com/g/konflux-announce/c/QILABamxDtA/m/5wK53cnlAgAJ